README.md

```diff
@@ -72,10 +72,44 @@ $ mix torch.gen.html Blog Post posts title:string body:text published_at:datetim
 The output would look like:
 
 ```bash
-Add the resource to your browser scope in lib/my_app_web/router.ex:
+* creating priv/templates/phx.gen.html/edit.html.heex
+* creating priv/templates/phx.gen.html/form.html.heex
+* creating priv/templates/phx.gen.html/index.html.heex
+* creating priv/templates/phx.gen.html/new.html.heex
+* creating priv/templates/phx.gen.html/show.html.heex
+* creating priv/templates/phx.gen.html/controller_test.exs
+* creating priv/templates/phx.gen.html/controller.ex
+* creating priv/templates/phx.gen.html/view.ex
+* creating priv/templates/phx.gen.context/access_no_schema.ex
+* creating priv/templates/phx.gen.context/context.ex
+* creating priv/templates/phx.gen.context/schema_access.ex
+* creating priv/templates/phx.gen.context/test_cases.exs
+* creating priv/templates/phx.gen.context/context_test.exs
+* creating lib/phx1_6_web/controllers/post_controller.ex
+* creating lib/phx1_6_web/templates/post/edit.html.heex
+* creating lib/phx1_6_web/templates/post/form.html.heex
+* creating lib/phx1_6_web/templates/post/index.html.heex
+* creating lib/phx1_6_web/templates/post/new.html.heex
+* creating lib/phx1_6_web/templates/post/show.html.heex
+* creating lib/phx1_6_web/views/post_view.ex
+* creating test/phx1_6_web/controllers/post_controller_test.exs
+* creating lib/phx1_6/blog/post.ex
+* creating priv/repo/migrations/20220201154756_create_posts.exs
+* creating lib/phx1_6/blog.ex
+* injecting lib/phx1_6/blog.ex
+* creating test/phx1_6/blog_test.exs
+* injecting test/phx1_6/blog_test.exs
```

Comment:
whats the difference between `creating` and `injecting`?

Reply:
That's the terminology Phx uses in it's generators.  Create is the actually creation of the file, inject is copying it into the correct directory location I believe.

Reply:
So files that aren't injected aren't placed onto the filesystem?

Reply:
I dont know.  Never looked into it. It's not something that Torch is doing specifically.  That output is from the underlying phoenix generators that Torch calls.